### ISSUE TITLE
Multiple crashes on iOS 8.4.1 (jailbroken)

### ISSUE THREAD

**Issue by Sample-User** [jasarien/Provenance#648]
Hi, first time reporting an issue in general.
First, thanks for your wonderful applications; I've recently installed the latest prebuild IPA from your repo on my iPhone 5 with iOS 8.4.1 (jailbroken EtasonJB) and while everything went smooth from this side I've encoutered a bunch of crash and problems in the application.
I have installed it from Filza directly from the iPhone without signing with AppSync and the application itself run just fine.
Major issue is that the menu button at the top when playing a game if pressed crash the entire application, and I think that because of this the emulator don't let me save (the only "fix" is enabling autosave and autoloading on launch from main menu) any game.
Another problem is when I close the Webserver, doing it simply crash the application.
Everything else is OK.
I cannot understand if is mainly a problem with the kind of installation/jailbreak or if these are simple bug in the application.
If required crash logs can be provided.
Thanks.

**Comment by jasarien** [comment]
Crash logs might be helpful, if they’re symbolicated (not so useful if they’re not). 

No-one else has reported crashes with the he menu button or webserver, but then no one else is still running iOS 8 or jailbroken. 

My first advice would be to upgrade to iOS 11 and forget the jailbreak, personally, but that’s up to whether or not you’re attached to the jailbreak or not.

**Comment by Sample-User** [comment]
Unfortunately looks like that every report is unsymbolicated (if with that you mean memory adresses instead of functions).

The only similar problems about menu button are these:
[https://github.com/jasarien/Provenance/issues/96](url)
[https://github.com/jasarien/Provenance/issues/477](url)

Max supported iOS version for my device is 10.3.3 however I think that I will stuck with that current version.
Don't worry to much about my problem because your advice it's perfectly understandable and legit.
Thanks for your time.




[Provenance_2018-01-10-194619_iPhone.txt](https://github.com/jasarien/Provenance/files/1620249/Provenance_2018-01-10-194619_iPhone.txt)

**Comment by jasarien** [comment]
I might be able to symbolicate it, if it’s the same binary you downloaded from github. I’ll have a try later.

**Comment by Sample-User** [comment]
Thanks again; take your time.
This is the version that I've downloaded: https://github.com/jasarien/Provenance/releases/tag/1.4

**Comment by JoeMatt** [comment]
I fixed a ton of iOS related issues on my new branch. There weren't really any checks for APIs that aren't support by iOS8. I changed all the target deployment targets to iOS 8 and a lot of the swift conversion caused Xcode to finally warn on API calls that wouldn't work on iOS and I wrapped all the ones it reported in if #available blocks. Should be working now on the N64+Swift branch though could use more testing
